docs/source/markdown/podman-auto-update.1.md.in

```diff
@@ -7,50 +7,47 @@ podman\-auto-update - Auto update containers according to their auto-update poli
 **podman auto-update** [*options*]
 
 ## DESCRIPTION
-**podman auto-update** looks up containers with a specified `io.containers.autoupdate` label (i.e., the auto-update policy).
+**podman auto-update** pulls down new container images and restarts containers configured for auto updates.
+To make use of auto updates, the container or Kubernetes workloads must run inside a systemd unit.
+After a successful update of an image, the containers using the image get updated by restarting the systemd units they run in.
+Please refer to `quadlet(5)` on how to run Podman under systemd.
 
-If the label is present and set to `registry`, Podman reaches out to the corresponding registry to check if the image has been updated. The label `image` is an alternative to `registry` maintained for backwards compatibility.
+To configure a container for auto updates, it must be created with the `io.containers.autoupdate` label or the `AutoUpdate` field in `quadlet(5)` with one of the following two values:
```

Comment:
My computer is beyond wonky.  I was commenting when my PC hung, and by the time I got back, Dan merged.

I'd replace `quadlet(5)` with Quadlet.  The first reference to quadlet(5) is OK, but it shouldn't be carried through out.  Also, you need to add `quadlet(5)` to the "See Also" list at the bottom.

Reply:
>  Also, you need to add quadlet(5) to the "See Also" list at the bottom.

That's what I did initially but it did not pass `make validate` as it's just a link to `podman-systemd.unit(5)`.